### ISSUE TITLE
🌍 Natural Earth earth features should have variable min_zoom

### ISSUE THREAD

**Issue by nvkelso** [tilezen/vector-datasource#1287]
tl;dr

- [ ] Earth features still need variable `min_zom`
- [x] Water layer features received variable `min_zoom` in v1.6

===

@sensescape found this one in the Refill LOD sprint:

- https://github.com/tilezen/vector-datasource/blob/master/yaml/water.yaml#L102-L128

```
- filter: {featurecla: Lake}
    min_zoom: 0
    output: {kind: lake}
    table: ne
```

All these are seeing to min_zoom 0, but we should be sourcing from ~the `scalerank` (or future~ Natural Earth's `min_zoom` values directly in the future).

Check also earth values.

<img width="718" alt="screen shot 2017-06-28 at 3 08 59 pm" src="https://user-images.githubusercontent.com/853051/27664225-3c6d95a6-5c1b-11e7-82f6-96252eb4d0bf.png">

<img width="663" alt="screen shot 2017-06-28 at 3 09 08 pm" src="https://user-images.githubusercontent.com/853051/27664226-3c8cb756-5c1b-11e7-95c1-08256fdbba3f.png">

**Comment by nvkelso** [comment]
Earth:

- https://github.com/tilezen/vector-datasource/blob/master/yaml/earth.yaml#L55-L58

```
  - filter: {gid: true}
    min_zoom: 0
    output: {kind: earth}
    table: ne
```

**Comment by nvkelso** [comment]
This was fixed in https://github.com/tilezen/vector-datasource/pull/1714 for v1.6 milestone.

**Comment by nvkelso** [comment]
Verified.

![image](https://user-images.githubusercontent.com/853051/56534187-03d8d400-650e-11e9-8f8d-5666b5941298.png)

![image](https://user-images.githubusercontent.com/853051/56534202-0a674b80-650e-11e9-9f51-76759b68a43d.png)

![image](https://user-images.githubusercontent.com/853051/56534213-0f2bff80-650e-11e9-935c-17a762b9e22a.png)
